src/Controller/BaseImportControllerTrait.php

```diff
@@ -76,14 +77,19 @@ protected function prepareSelectFileView(FormInterface $form): Response
         );
     }
 
-    protected function prepareMatrixEditView(Matrix $matrix, EntityManagerInterface $entityManager): Response
+    protected function prepareMatrixEditView(Matrix $matrix, EntityManagerInterface $entityManager, bool $manualSubmit = false): Response
     {
+        $form = $this->createMatrixForm($matrix, $entityManager);
```

Comment:
Please move `$form` and place it before this function. This function should get `FormInterface` as a parameter. This change is needed to make form error visible.

Reply:
Can you update a readme?

Reply:
done